Golden_Path.postman_collection.json

```diff
@@ -1,7 +1,7 @@
 {
 	"info": {
-		"_postman_id": "5591457e-465c-43be-bd9d-4863ebd7747e",
-		"name": "Golden_Path",
+		"_postman_id": "6853006f-5da9-47e9-8c5b-f54b5e1bd15c",
+		"name": "Golden_Path Copy",
```

Comment:
Copy?

Reply:
Updated default file name in 2nd commit - exporting issue.